cmd/install/assets/hypershift_operator.go

```diff
@@ -1183,6 +1183,11 @@ func (o HyperShiftOperatorClusterRole) Build() *rbacv1.ClusterRole {
 				Resources: []string{"virtualmachineinstances/addvolume", "virtualmachineinstances/removevolume"},
 				Verbs:     []string{rbacv1.VerbAll},
 			},
+			{ // This allows the kubevirt csi driver to hotplug volumes to KubeVirt VMs.
+				APIGroups: []string{"subresources.kubevirt.io"},
+				Resources: []string{"virtualmachines/addvolume", "virtualmachines/removevolume"},
+				Verbs:     []string{rbacv1.VerbAll},
```

Comment:
Does this really need all or is get just fine?

Reply:
I am pretty sure it only needs "update", good call. Not sure why it uses * prior to this